tests/src/test/scala/ha/ShootComponentsTests.scala

```diff
@@ -67,13 +70,13 @@ class ShootComponentsTests
   val controller0DockerHost = WhiskProperties.getBaseControllerHost()
   val couchDB0DockerHost = WhiskProperties.getBaseDBHost()
 
-  val dbProtocol = WhiskProperties.getProperty(WhiskConfig.dbProtocol)
+  val dbConfig = loadConfigOrThrow[CouchDbConfig](ConfigKeys.couchdb)
+  val dbProtocol = dbConfig.protocol
   val dbHostsList = WhiskProperties.getDBHosts
-  val dbPort = WhiskProperties.getProperty(WhiskConfig.dbPort)
-  val dbUsername = WhiskProperties.getProperty(WhiskConfig.dbUsername)
-  val dbPassword = WhiskProperties.getProperty(WhiskConfig.dbPassword)
-  val dbPrefix = WhiskProperties.getProperty(WhiskConfig.dbPrefix)
-  val dbWhiskAuth = WhiskProperties.getProperty(WhiskConfig.dbAuths)
+  val dbPort = dbConfig.port
+  val dbUsername = dbConfig.username
+  val dbPassword = dbConfig.password
+  val dbWhiskAuth = dbConfig.databases.get("WhiskAuth").get
```

Comment:
Can this be `dbConfig.databaseFor[WhiskAuth]`?

Reply:
Problem here is `WhiskAuth` is scoped to `core` package and test lives outside of that package. So need to use the string value explicitly